tests/test_json.py

```diff
@@ -96,6 +96,9 @@ class News(db.Model):
         profile = db.Column(JSON(), nullable=False, server_default='{}')
         visits = db.IntegerProperty(default=0)
 
+        def lookup(self):
+            return True
```

Comment:
This should be discouraged, right? If so, let's not set a bad example in the test.